test/test-lib-mysql.sh

```diff
@@ -129,18 +129,24 @@ function test_mariadb_integration() {
 # Check the imagestream
 function test_mariadb_imagestream() {
   case ${OS} in
-    rhel7|centos7|rhel8) ;;
+    rhel7|centos7|rhel8|rhel9) ;;
     *) echo "Imagestream testing not supported for $OS environment." ; return 0 ;;
   esac
   local tag="-el7"
   if [ "${OS}" == "rhel8" ]; then
     tag="-el8"
+  elif [ "${OS}" == "rhel9" ]; then
+    tag="-el9"
   fi
   ct_os_test_image_stream_template "${THISDIR}/imagestreams/mariadb-${OS%[0-9]*}.json" "${THISDIR}/examples/mariadb-ephemeral-template.json" mariadb "-p MARIADB_VERSION=${VERSION}${tag}"
 }
 
 function test_mariadb_template() {
-  if [[ "${OS}" =~ rhel7 ]] || [[ "${OS}" =~ centos7 ]] || [[ "${OS}" =~ rhel8 ]] ; then
+  if [[ "${OS}" =~ rhel9 ]] && [[ "${VERSION}" == "10.3" ]]; then
+    echo "Testing mariadb_template is not supported for RHEL9 and version 10.3"
```

Comment:
Why?

Reply:
mariadb-10.3 on RHEL9 is not supported.

Reply:
We should change this condition as discussed in person.

Reply:
resolved by https://github.com/sclorg/mariadb-container/pull/190/commits/a27dcae273c18f134e223b17ebc45e15a798a93d